src/OrleansRuntime/Scheduler/SchedulingContext.cs

```diff
@@ -147,6 +147,23 @@ public override string ToString()
                     return "";
             }
         }
+        public string DetailedStatus()
+        {
+            switch (ContextType)
+            {
+                case SchedulingContextType.Activation:
+                    return Activation.ToDetailedString(true);
+
+                case SchedulingContextType.SystemTarget:
+                    return SystemTarget.ToDetailedString();
+
+                case SchedulingContextType.SystemThread:
+                    return String.Format("DispatcherTarget{0}", DispatcherTarget);
+
+                default:
+                    return "";
```

Comment:
ContextType.ToString() on default?  If we add more context types, at least this will tell use which ones don't have detailed reports.